marconi-chain-index/src/Marconi/ChainIndex/Indexers.hs

```diff
@@ -138,13 +138,13 @@ datumWorker Coordinator{_barrier, _channel} path = do
 utxoWorker_
   :: (Utxo.UtxoIndexer -> IO ())  -- ^ callback function used in the queryApi thread, needs to be non-blocking
   -> Utxo.Depth
-  -> Maybe TargetAddresses                       -- ^ Target addresses to filter for
+  -> Maybe TargetAddresses        -- ^ Target addresses to filter for
   -> Coordinator
   -> TChan (ChainSyncEvent (BlockInMode CardanoMode))
   -> FilePath
   -> IO (IO (), MVar Utxo.UtxoIndexer)
 utxoWorker_ callback depth maybeTargetAddresses Coordinator{_barrier} ch path = do
-  ix <- Utxo.open path depth
+  ix <- Utxo.open path depth True -- open SQLite with depth=depth and perform SQLite vacuum
```

Comment:
if this change is accepted, I'd like to added to the CLI parameters for both:
- marconi-chain-index
- marconi-sidechain
on a separate PR